CMakeLists.txt

```diff
@@ -10,6 +10,9 @@ set_property( GLOBAL PROPERTY USE_FOLDERS ON )
 # Use C90.
 set( CMAKE_C_STANDARD 90 )
 set( CMAKE_C_STANDARD_REQUIRED ON )
+set( CMAKE_THREAD_PREFER_PTHREAD ON )
+
+find_package(Threads REQUIRED)
```

Comment:
Does it still work if we move this to `demos/mqtt/mqtt_demo_basic_tls/CMakeLists.txt` or `demos/CMakeLists.txt` (preferably the first).

Reply:
Appears so.